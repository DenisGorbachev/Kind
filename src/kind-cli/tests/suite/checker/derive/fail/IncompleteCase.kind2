record User {
    constructor new
    name : U60
    ttt : U60
    e : U60
}

Main : U60
Main = 
    let User.new (ttt = e) e .. = User.new 2 4 1
    let User.new (ttt = f) name = User.new 6 7 3
    e
