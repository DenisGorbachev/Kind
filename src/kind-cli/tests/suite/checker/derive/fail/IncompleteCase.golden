   ERROR  The case is not covering all the values inside of it!

      /--[tests/suite/checker/derive/fail/IncompleteCase.kind2:12:9]
      |
   11 |        let User.new (ttt = e) e .. = User.new 2 4 1
   12 |        let User.new (ttt = f) name = User.new 6 7 3
      |            v-------
      |            \This is the incomplete case
   13 |        e

      Hint: Need variables for 'e'

